Cargo.toml

```diff
@@ -75,6 +75,7 @@ actix-session = ["paperclip-core/actix-session"]
 actix-files = ["paperclip-core/actix-files"]
 chrono = ["paperclip-core/chrono"]
 rust_decimal = ["paperclip-core/rust_decimal"]
+url_feature = ["paperclip-core/url"]
```

Comment:
should this simply be `url` to keep it like the other ones?

Reply:
A feature can't have the same name as a dependency:
https://github.com/wafflespeanut/paperclip/pull/334/files/c1e400c376c96eee52f6407685a8001a4ecbf260#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R40

Reply:
interesting so how does it work for the other ones, eg uuid?

Reply:
Code was changes to use  `url` as a feature name and renamed `url` dependency as `url_dep`.